types/country-telephone-data/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for country-telephone-data 0.6
+// Project: https://github.com/mukeshsoni/country-telephone-data#readme
+// Definitions by: Kyle Hensel <https://github.com/k-yle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Country {
```

Comment:
Does it miss `priority` property?
https://github.com/mukeshsoni/country-telephone-data/blob/master/country_telephone_data.js#L350

Reply:
oops, yes. Added it